pkg/api/types.go

```diff
@@ -302,24 +302,43 @@ type DcosConfig struct {
 	RegistryPass            string `json:"registryPassword,omitempty"`
 }
 
+// NetworkAccessProfile describes inbound traffic rules for network access.
+type NetworkAccessProfile struct {
+	SourceAddressPrefix        string   `json:"sourceAddressPrefix"`
```

Comment:
Let's get rid of the "single" `SourceAddressPrefix` property. Reading the templates above, it looks like we have two paths: user specified one rule, or a bunch of rules, if the former, apply it, if the latter, enumerate and apply them all. I would prefer we just do the enumeration no matter what, so if we want to specify a single source address prefix (or source port range, or dest address prefix, or dest port range) we the config simply declares an array with one item.